src/backends/index.ts

```diff
@@ -47,7 +47,7 @@ export function newCeleryBackend(
     return new RedisBackend(CELERY_BACKEND, CELERY_BACKEND_OPTIONS);
   }
 
-  if (brokerProtocol === "amqp") {
+  if (brokerProtocol.startsWith("amqp")) {
```

Comment:
I guess we should use `indexOf` for strict check to prevent unexpected error.

https://github.com/actumn/celery.node/blob/e646b545337feb9b8b762b9d94f65a620da3494c/src/kombu/brokers/index.ts#L33-L39

So what about writing like below?

```suggestion
  if (['amqp', 'amqps'].indexOf(brokerProtocol) === 1) {
```

Reply:
Good point re strict, I think it'd be `> -1` though, no?

```suggestion
  if (['amqp', 'amqps'].indexOf(brokerProtocol) > -1)) {
```

Reply:
Yeah, that's true. Sorry for making you confused.